recipes/rmm/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "rmm" %}
+{% set version = "21.12.00" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 71802422a04f2e0a828f7b7ec77e5cd1d24361d6dc5bf60a534659e4c2e08a44
+  folder: rmm
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version ==  "None") or (aarch64) or (py < 37) or (py >= 39)]
+
+  script_env:
+    - INSTALL_PREFIX=${PREFIX}
+
+  script:
+    - cd rmm/python
+    - {{ PYTHON }} -m pip install . --no-deps -vv
+
+  missing_dso_whitelist:
+    - '*/libcuda.*'  # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+  host:
+    - librmm ={{ version }}
+    - python
+    - pip
+  run:
+    - python
+    - numba >=0.49
+    - numpy
+    - {{ pin_compatible('cudatoolkit', max_pin='x', lower_bound='11.2') }} # cudatoolkit >=11.2,<12.0.0{{ pin_compatible('cudatoolkit', max_pin='x', lower_bound='11.2') }} # cudatoolkit >=11.2,<12.0.0
+    - pip
+    - setuptools
+    - cython >=0.29,<0.30
+    - spdlog >=1.8.5,<2
```

Comment:
These do not make sense (they're build time dependencies, not runtime) and were not what I suggested....

Reply:
Mistakenly done that. Reverted.